apps/src/lib/client/tx.rs

```diff
@@ -57,6 +65,129 @@ pub async fn aux_signing_data<'a>(
     Ok(signing_data)
 }
 
+// Sign the given transaction using a hardware wallet as a backup
+pub async fn sign<'a>(
+    context: &impl Namada<'a>,
+    tx: &mut Tx,
+    args: &args::Tx,
+    signing_data: SigningTxData,
+) -> Result<(), error::Error> {
+    // Setup a reusable context for signing transactions using the Ledger
```

Comment:
I prefer that we use the Ledger app only explicitly, when the user has instructed us to do so, by either:
- passing some flag (such as `--use-device`), or
- using a key which we know (from the saved data) to have been generated with the Ledger

Reply:
I've implemented option 1, `--use-device`, to trigger hardware wallet signing.